Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -146,7 +146,7 @@ private void AddSettingsWithout<T>(StringBuilder codeBuilder, JObject settings,
             var indentation = string.Join(string.Empty, Enumerable.Repeat(" ", indentationDepth));
 
             var filteredSettings = ((IEnumerable<KeyValuePair<string, JToken>>)settings)
-                .Where(setting => !setting.Key.EqualsOrdinal(typeof(T).Name));

```

Comment:
It may make sense to keep the extension method in place, but decorate it with an `Obsolete` attribute motivating the usage of `==`, so that no-one will try to do the same changes again in the future.

Reply:
This is not in released code yet, actually, only added here: https://github.com/Lombiq/Helpful-Libraries/pull/102/files/3eccd97b413f155aab3efe704b48bae87b2bf18e..40112b9e091b02512e9bdad4a523b06e68cc340c#diff-9395a517626951194a7a9e325305660b0db18ebceffc0823f5c64d638ee105fa